data/starts.txt

```diff
@@ -0,0 +1,203 @@
+	set "hai space start"
+	# need to set it here to not have it trigger alongside the invisible mission
+	set "First Contact: Hai: offered"
+	set "Intro [0]: offered"
+
+
+
+conversation "hai intro"
+	scene "scene/hai shipyard"
+	`You've never been to the shipyard before, and going to the North Pole already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
+	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
+	scene "thumbnail/hai aphid"
+	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`
+	`	You have, after all, heard tales of pirates and quick money from the many newcomers to Hai space - for those willing to take big risks, that is.`
+	`	Alternatively, up north you can go and meet the "Unfettered Hai." Maybe they would come to like you, if you were to show them the strength and cunning they respect.`
+	choice
+		`	(Go to the bank.)`
+	scene "scene/hai bank"
+	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`
+	`	To your disappointment, the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life.`
+	`	You've heard of new captains stripping down their ships to recoup the initial buy-in, which is crossing your mind as you see the financial burden you're about to take on. As you wait for the paperwork to be processed, you passively scan over an Aphid pamphlet in the office. The pricey shield regenerator could be traded in if dangerous areas are avoided... and perhaps even the anti-missile system could be downgraded or outright removed... one of the Sand Cells could be sold back to make more space for cargo...`
+	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
+	name
+	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
+	choice
+		`	(Go to the shipyard.)`
+			decline
+		`	(Go see what the captain has to offer first.)`
+	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully he has not yet sold the Grasshopper.`
```

Comment:
Again, since you're from the equator where the Hai don't usually frequent, you'll probably differentiate this captain as being a Hai. ~~It was also a little confusing.~~
```suggestion
	`	You head to the spaceport and easily find the captain, a Hai, selling one of his Centipedes to another Hai. Thankfully it looks like he has not yet sold the Grasshopper.`
```
I was also confused what the captain was doing. Was he idling around and waiting for buyers, or did he already have some? I like the second option.

Reply:
you want to add other ppl betting against you haha?